homework.md

```diff
@@ -20,48 +20,217 @@ CREATE TABLE stations
 );
 ```
 
-1) Write a query that fills out this table. Try your best to pick the correct station name for each ID. You may have to make some manual choices or editing based on the inconsistencies we've found. Do try to pick the correct name for each station ID based on how popular it is in the trip data.
+1. Write a query that fills out this table. Try your best to pick the correct station name for each ID. You may have to make some manual choices or editing based on the inconsistencies we've found. Do try to pick the correct name for each station ID based on how popular it is in the trip data.
 
-Hint: your query will look something like 
+### Question 1 Answer:
 
-``` sql
-INSERT INTO stations (SELECT ... FROM tripe);
+```sql
+CREATE TABLE station_popularity
+(
+    id INT,
+    name VARCHAR(100),
+    popularity integer
+);
+
+INSERT INTO station_popularity
+SELECT Max(from_station_id) as station_id, from_station_name as station_name, count(from_station_name) as popularity
+FROM trips
+GROUP BY station_name;
+
+INSERT INTO station_popularity
+SELECT Max(to_station_id) as station_id, to_station_name as station_name, count(to_station_name) as popularity
+FROM trips
+GROUP BY station_name;
```

Comment:
I see what you're trying to do here; something about the particular way you've done this though, is that you end up with duplicate name entries coming from the `to_*` set and the `from_*` set in the `station_popularity` table. It works for the purposes of the exercise, but can you think of a way you could have consolidated station names from both directions?